src/vivarium_gates_mncnh/components/observers.py

```diff
@@ -65,3 +65,18 @@ def register_observations(self, builder: Builder) -> None:
     def format(self, measure: str, results: pd.DataFrame) -> pd.DataFrame:
         new_births = results[list(self.COL_MAPPING)].rename(columns=self.COL_MAPPING)
         return new_births
+
+
+class ANCObserver(Observer):
```

Comment:
Is this really what we want to observe here? I would have thought we'd want to get count data, not simulant-specific data.